src/app/state/blogpost/actions/post-api.actions.ts

```diff
@@ -0,0 +1,9 @@
+import { Post } from '../../../blogpost/post';
+
+/* NgRx */
+import { createAction, props } from '@ngrx/store';
+
+export const loadPostsSuccess = createAction(

```

Comment:
We used to keep related actions in a single file (and prefix with _action_) :
actionLoadPosts
actionLoadPostsSuccess
actionLoadPostsError

in that manner it is easier to maintain related actions..
and prefix makes autocomplete in IDE simplier - as large projects will have hundreds of actions. 
after that you can type: _aclopo_ or similar part of action and ide will filter out most actions:
this.store.dispatch(_ac_lo_po_ctrl+space

Reply:
so suggestion is to keep actions for APIs and actions for components in the same file, or I am misunderstanding?

Reply:
usually it's one file per feature, but feel free to organize actions more gradually if that suits better